deploy/eks/kiam-server-deplyoyment.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
why not daemonset?
https://github.com/uswitch/kiam/blob/master/deploy/server.yaml#L3

Reply:
why would you use a daemonset? a pod on EVERY node in the cluster to serve this?